spec/features/user_can_view_all_horses_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../spec_helper'
+
+RSpec.describe "when user visits horses path" do
+  it "they view all horses" do
+    Jockey.create(name: "Joey")
+    Breed.create(name: "Palomino")
+
+    horse1 =  Horse.create(name: "Penelope", age: 29, total_winnings: 34000, breed_id: 1, jockey_id: 1)
```

Comment:
If you wanted to create a horse with *Joey* as the jockey, you could also save the *instance* Joey to a variable and then call create on Joey's horses. Believe that syntax is slightly preferred, though certainly different people will have their opinions. Additionally, AR is smart enough to let us pass instances instead of ids (look at what I'm doing in the following example with `palomino`). For example:

```ruby
joey         = Jockey.create(name: "Joey")
palomino = Breed.create(name: "Palomino")

horse_1 = joey.horses.create(name: "Penelope", age: 29, total_winnings: 34000, breed: palomino)
```